src/main/java/com/razorpay/ApiUtils.java

```diff
@@ -24,7 +25,10 @@
 
   static void createHttpClientInstance(boolean enableLogging) throws RazorpayException {
     if (client == null) {
-      client = new OkHttpClient.Builder().build();
+      client = new OkHttpClient.Builder()
+              .readTimeout(60, TimeUnit.SECONDS)
+              .writeTimeout(60, TimeUnit.SECONDS)
```

Comment:
@captn3m0 By default each have a value of 10 sec. [Check this](https://github.com/square/okhttp/blob/okhttp_25/okhttp/src/main/java/com/squareup/okhttp/OkHttpClient.java#L190)